nat-traversal/get_relay_signature_key.go

```diff
@@ -0,0 +1,45 @@
+package nattraversal
+
+import (
+	nex "github.com/PretendoNetwork/nex-go"
+	nat_traversal "github.com/PretendoNetwork/nex-protocols-go/nat-traversal"
+)
+
+func getRelaySignatureKey(err error, client *nex.Client, callID uint32) {
+	server := commonNATTraversalProtocol.server
+	rmcResponseStream := nex.NewStreamOut(server)
+
+	rmcResponseStream.WriteInt32LE(0)
+	dateTime := nex.NewDateTime(0)
+	dateTime.Now()
+	rmcResponseStream.WriteDateTime(dateTime)
+	rmcResponseStream.WriteString("")
+	rmcResponseStream.WriteUInt16LE(0)
+	rmcResponseStream.WriteInt32LE(0)
+	rmcResponseStream.WriteUInt32LE(0x55d01)
```

Comment:
We know the contents of some of this fields, we don't need to hardcode it

```suggestion
	rmcResponseStream.WriteInt32LE(0) // relayMode
	dateTime := nex.NewDateTime(0)
	dateTime.UTC()
	rmcResponseStream.WriteDateTime(dateTime) // currentUTCTime
	rmcResponseStream.WriteString("") // address
	rmcResponseStream.WriteUInt16LE(0) // port
	rmcResponseStream.WriteInt32LE(0) // relayAddressType
	rmcResponseStream.WriteUInt32LE() // gameServerID. Allow the implementation to set this field with a handler (something like `SetGameServerID()`?)
```

Also, are the address and port *really* empty? Aren't they the address and port of the client or something?

Reply:
It's supposed to be the address of the relay server. We don't have an implementation of the relay server, so the response tells the client to not use it.

Reply:
How much information do we have about the relay servers? Should I add this to the todo list?

Reply:
Honestly, not much info. I wouldn't make them a priority for now.